public/style.css

```diff
@@ -12,10 +21,63 @@
   overflow: auto;
 }
 
+h1 {
+  margin-bottom: 15%;
+  color: #455eb5;
+  font-size: 3rem;
+}
+
+button {
+  margin-top: 10%;
+  background-image: linear-gradient(
+    92.88deg,
+    #455eb5 9.16%,
+    #5643cc 43.89%,
+    #673fd7 64.72%
+  );
+  border-radius: 0.7rem;
+  border-style: none;
+  box-sizing: border-box;
+  color: #ffffff;
+  cursor: pointer;
```

Comment:
Please double-check if you need this line. Since the `button` element already has cursor as pointer.

Reply:
doesn't work if i remove this condition